ui/components/app/edit-gas-fee-popover/network-status/status-slider/status-slider.js

```diff
@@ -15,38 +16,60 @@ const GRADIENT_COLORS = [
   '#C54055',
 ];
 
+const StatusInfo = {
```

Comment:
[nit] What do you feel about using `statusInfo` or `STATUS_INFO` for the name? `StatusInfo` could be misinterpreted as a class or component.

Reply:
I changed this